events/2025-03-10-hts-workshop-freiburg.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: event
+title: "Workshop on high-throughput sequencing data analysis with Galaxy"
+
+# external: "https://galaxyproject.org/events/2025-03-10-galaxy-workshop-freiburg/"
+
+description: |
+    This course introduces scientists to the data analysis platform Galaxy. The course is a beginner course; no programming skills are required.
+
+date_start: 2025-03-10
+date_end: 2025-03-14
+
+cost: free
+audience: Scientists with no or little Galaxy experience who want to analyse sequencing data.
+contact_email: schneidd@informatik.uni-freiburg.de
+# async: false
+mode: onsite
+
+registration:
+  link: https://forms.gle/cC9ynBrypEWwNA9u8
+  deadline: 2025-02-14
+
+contributions:
+  organisers: [Sch-Da, teresa-m]
+  instructors: [pavanvidem, Nilchia, teresa-m, wm75, EngyNasr, paulzierep]
+  funding: [eurosciencegateway, deNBI, nfdi4plants]
+
+location:
+  geo:
+    lat: 47.993460
+    lon: 7.845110
+  name: University Freiburg
+  # address: Werthmannstrasse 4
+  # postcode: 79104
```

Comment:
i would suggest to include these, or is there a reason they are commented out?

Reply:
I think when I tried last time I got an error while building the site locally. But I will check again. Thanks for the hints.

Reply:
please let me know what the error is if you encounter it

Reply:
Sorry, only an indentation error from my side. I added the Street now. 
About the geo location, I am not sure how to do this.

Reply:
>  About the geo location, I am not sure how to do this.

yes, don't add the geo. You don't have to add it / do anything with it at all, please ignore it completely :)

Reply:
Hi @hexylena, thanks for your help first. I did not remove the geo, but now I am losing the location in my local build, which would be nice to have. 
![Screenshot from 2024-10-15 14-06-34](https://github.com/user-attachments/assets/742b6f6e-a534-4976-892f-f23921a5abfd)

Reply:
Should this also automatically apply using the address? If not I would like to keep the locations

Reply:
@hexylena: Thanks for the explanation. Now I understand better what to do. Next time I will add the address and level to your automatication to add the geo location.